src/test/java/com/ft/methodecontentcollectionmapper/validation/ContentCollectionValidatorTest.java

```diff
@@ -14,23 +16,84 @@
 
 @RunWith(MockitoJUnitRunner.class)
 public class ContentCollectionValidatorTest {
+
     private ContentCollectionValidator contentCollectionValidator;
 
     @Before
     public void setUp() {
         contentCollectionValidator = new ContentCollectionValidator();
     }
 
+
+    @Test(expected = ValidationException.class)
+    public void thatNullMainUuidThrowsException() {
+        final EomFile eomFile = new EomFile(null, null, null, null, null, null, null, null);
```

Comment:
Does this class EomFile have a builder which you could use here with default values?